apps/frontend/src/libs/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from "./styles.module.css";
+
+interface Properties {
+	name?: string | undefined;
```

Comment:
`name?: string | undefined;` can be simplified to `name?: string;` since undefined is already implied by the optional modifier ?

Reply:
@sandrvvu with the new typescript versions, `name?` and `name: undefined | string` are different types. The first one is for case where you don't pass property at all and `name={undefined}` is not allowed in this case. The second one requires you to pass the property, but it can be undefined. So you can do this `name={undefined}`, but not to omit the name property